outwatch/yarn.lock

```diff
@@ -2,3929 +2,538 @@
 # yarn lockfile v1
```

Comment:
why remove it?

Reply:
Oh, that was not my intention.

Reply:
It is not removed, but much smaller. I do not know why though :D

Reply:
where is it then?

Reply:
It is gone :)

Reply:
It seems like the require `snabbdom/h` pulled that all in? If add hfunction back in, I get the old huge lock file, without the hfunction it gets smaller :parasol_on_ground: